workspaces/tech-insights/plugins/tech-insights-backend/src/service/router.ts

```diff
@@ -67,6 +81,16 @@ export interface RouterOptions<
    * Implementation of Winston logger
    */
   logger: LoggerService;
+
+  /**
+   * Implementation of PermissionsService
+   */
+  permissions: PermissionsService;
+
+  /**
+   * Implementation of HttpAuthService
+   */
+  httpAuth: HttpAuthService;
```

Comment:
With this change will this still work with the legacy backend? Might make sense to remove support for the legacy backend as well. Would have been nice to do a deprecation cycle but as you already have a breaking change it might make sense to include it.